app/assets/stylesheets/application.scss

```diff
@@ -185,3 +176,26 @@ h1 a:hover {
   color: #6FDF52;
   text-align: center;
 }
+// Footer
+
+.page-wrap {
+	margin-bottom: -142px;
+  min-height: 120%;
+}
+
+.page-wrap:after {
+  content: "";
+  display: block;
+}
+
+.site-footer,
+.page-wrap:after {
+  height: 142px;
+}
+
+footer {
+  background-color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.